cirq-google/cirq_google/engine/engine_processor.py

```diff
@@ -119,12 +330,17 @@ def get_device(self, gate_sets: Iterable[SerializableGateSet]) -> cirq.Device:
         spec = self.get_device_specification()
         if not spec:
             raise ValueError('Processor does not have a device specification')
-        return serializable_device.SerializableDevice.from_proto(spec, gate_sets)
+        if not all(isinstance(gs, SerializableGateSet) for gs in gate_sets):
+            raise ValueError('All gate_sets must be SerializableGateSet currently.')
+        return serializable_device.SerializableDevice.from_proto(
+            spec, cast(Iterable[SerializableGateSet], gate_sets)
+        )
 
     def list_calibrations(
         self,
-        earliest_timestamp_seconds: Optional[int] = None,
-        latest_timestamp_seconds: Optional[int] = None,
+        earliest_timestamp: Optional[Union[datetime.datetime, datetime.date, int]] = None,
+        latest_timestamp: Optional[Union[datetime.datetime, datetime.date, int]] = None,
+        **kwargs,
     ) -> List[calibration.Calibration]:
```

Comment:
Does this need a docstring update or deprecation warning ?

Reply:
Fixed with `@deprecated_parameter`.